lib/urlwatch/reporters.py

```diff
@@ -600,11 +600,29 @@ def submit(self):
 
         return result
 
+    @staticmethod
+    def _format_body(text: str) -> str:
+        return "```diff\n{}\n```".format(
+            text.translate(str.maketrans({'`': r'\`', '\\': r'\\'})))
+
     def submitToTelegram(self, bot_token, chat_id, text):
         logger.debug("Sending telegram request to chat id:'{0}'".format(chat_id))
-        result = requests.post(
-            "https://api.telegram.org/bot{0}/sendMessage".format(bot_token),
-            data={"chat_id": chat_id, "text": text, "disable_web_page_preview": "true"})
+
+        data = {"chat_id": chat_id,
+                "text": text,
+                "disable_notification": self.config['disable_notification'],
```

Comment:
Since this might not be set, you have to use:
```
self.config.get('disable_notification', False)
```

(or `'silent'`, see other comments)

Reply:
added